lib/lita/handlers/jenkins.rb

```diff
@@ -12,27 +12,37 @@ class Jenkins < Handler
         'jenkins list <filter>' => 'lists Jenkins jobs'
       }
 
-      route /j(?:enkins)? b(?:uild)? ([\w\.\-_ ]+)/i, :jenkins_build, command: true, help: {
+      route /j(?:enkins)? b(?:uild)? ([\w\.\-_ ]+)(, (.+))?/i, :jenkins_build, command: true, help: {
         'jenkins b(uild) <job_id or job_name>' => 'builds the job specified by ID or name. List jobs to get ID.'
       }
 
-      def jenkins_build(response)
-        job = find_job(response.matches.last.last)
+      def jenkins_build(response, empty_params = false)
```

Comment:
`empty_params` looks fishy to me. You shouldn't need an extra variable just to keep track if something is empty...just check if the thing is empty.

One thing that might help the code here is [keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments). It's a new feature in ruby 2. It will let you name your parameters so that when you pass them in you know exactly what the method expects (e.g. `process_images(image_name: i_name, timestamp: DateTime.now)`), you can be used for optional parameters, and can be used to require certain parameters.

Please check it out and let me know if you have questions on how to apply it to this PR.